types/state-change-events.d.cts

```diff
@@ -0,0 +1,144 @@
+type ErrorSource = {
+	isDependency: boolean;
+	isWithinProject: boolean;
+	file: string;
+	line: number;
+};
+
+type SerializedErrorBase = {
+	message: string;
+	name: string;
+	originalError: unknown;
+	stack: string;
+};
+
+type AggregateSerializedError = SerializedErrorBase & {
+	type: 'aggregate';
+	errors: SerializedError[];
+};
+
+type NativeSerializedError = SerializedErrorBase & {
+	type: 'native';
+	source: ErrorSource | undefined;
+};
+
+// eslint-disable-next-line @typescript-eslint/naming-convention
```

Comment:
a new rule started complaining about this, ava is always stylized AVA so I think it's ok to disable here?

Reply:
I'd like to stick with the rule here. Will push up a commit.